vimrc

```diff
@@ -36,6 +36,7 @@ set ignorecase
 set smartcase
 set wildignore+=*.pyc,*.o,*.class,*.lo,.git,vendor/*,node_modules/**,bower_components/**,*/build_gradle/*,*/build_intellij/*,*/build/*,*/cassandra_data/*
 set tags+=gems.tags
+set backupcopy=yes " Setting backup copy preserves file inodes, which are needed for Docker file mounting
```

Comment:
Our vim settings default `backup copy` to `auto`, but the default vi setting is in fact `yes`.